### ISSUE TITLE
[3.4] Bottom save button on records is different to the one on the side

### ISSUE THREAD

**Issue by Pinpickle** [bolt/bolt#7066]
In the editing page for a record, the save button at the bottom has different UX to that on the side widget. This is especially prevalent with singleton records.

Details
-------

 - Relevant Bolt Version: 3.4-beta2
 - Install type: Composer Install
 - PHP version: 7.1
 - Used web server: Apache

Reproduction
------------

### Steps to reproduce

Create a new Bolt site, add any content type (make it a singleton for dramatic effect). For example

```
home_pages:
    name: Home Page
    singular_name: Home Page
    fields:
        about_heading:
            label: About Heading
            type: text
    singleton: true
    default_status: published
    sort: -id
    icon_many: "fa:home"
```

### Expected result

The save button on the bottom behaves exactly the same as the one on the widget on the right.

### Actual result

The one on the bottom is different, it allows me to create new singleton records with the dropdown arrow, and its label is the `singular_slug` for the contenttype, not its `singular_name`.

![image](https://user-images.githubusercontent.com/3238878/31397097-fae39482-addc-11e7-8b24-41b900b949de.png)

**Comment by GawainLynch** [comment]
Top marks! I was waiting to see how long until someone noticed that :wink:

**Comment by GawainLynch** [comment]
:hamster: power fixed this in #7068
